tests/integration/cluster.go

```diff
@@ -1275,7 +1275,11 @@ func NewClusterV3(t testing.TB, cfg *ClusterConfig) *ClusterV3 {
 		for _, m := range clus.Members {
 			client, err := NewClientV3(m)
 			if err != nil {
-				t.Fatalf("cannot create client: %v", err)
+				if t != nil {
+					t.Fatalf("cannot create client: %v", err)
```

Comment:
hmm, `t` is ever `nil`?

Reply:
// t might be nil in case of Examples and clusters created per test-suite.